### ISSUE TITLE
focused window hidden if it has been movetoworkspaced to a fullscreen workspace

### ISSUE THREAD

**Issue by Dickby** [hyprwm/Hyprland#4172]
### Hyprland Version

Hyprland, built from branch makepkg at commit b1b8d732e64ecf527baef010ad2f28ed3b8c4ac1 dirty (toplevelexport: fix missed pmonitor set for opengl). Tag: v0.33.1-47-gb1b8d732

### Bug or Regression?

Bug

### Description

When moving a window to a worspace that has a fullscreen window, the moved window will still be focused, but hidden behind the fullscreen window.
Not shure if it's important, but `follow-mouse` is set to 1

### How to reproduce

Fullscreen a window,
switch to another workspace and move a window to the fullscreened workspace with `movetoworkspace`.

**Comment by AxerTheAxe** [comment]
This is not a bug. You can alter the behavior by changing the following setting:

`
misc {
  new_window_takes_over_fullscreen = 1
}
`
More information can be found [here](https://wiki.hyprland.org/Configuring/Variables/#misc).

**Comment by Dickby** [comment]
@AxerTheAxe : 
No, this is a bug, and changing new_window_takes_over_fullscreen doesn't change this.
That option is for windows that get newly created on a already fullscreen workspace.
If you reproduce the problem with the steps i listed, you will get to an totally unusable state, where one window is shown fullscreen, but another window is focuse beneath it.

**Comment by AxerTheAxe** [comment]
Strange, new_window_takes_over_fullscreen fixes the issue for me when using the steps you provided. I will look into it and see what I can find.

**Comment by ripytide** [comment]
I also experience this bug, a similar and possibly related bug is that when moving a fullscreen window to a workspace it now stays in fullscreen whereas back in `v0.30.0` it made the new workspace not fullscreen. `v0.30.0` seems to be the last version that worked like this.

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
